common/lib/api-client/transformers/move.transformer.js

```diff
@@ -9,3 +9,43 @@ module.exports = function moveTransformer(data) {
 
   data._is_youth_move = isYouthMove
 }
+
+const addImportantEvents = data => {
+  // timeline_events and important_events are mutually exclusive at the BE
+  // copy any event with a non-default classification to .important_events
+  // maintaining the same order
+  if (data.timeline_events && data.timeline_events.length) {
```

Comment:
Could this be a little safer and check that important events doesn't exist before creating/overwriting it?

Reply:
Switching the check `!data.important_events? && data.timeline_events?.length`

However
a) They are mutually exclusive.
b) Even if both properties were returned, the routine would just filter the timeline_events to create exactly the same important_events. Think about it...